javav2/example_code/stepfunctions/src/test/java/StepFunctionsTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+import com.example.stepfunctions.*;
+import org.junit.jupiter.api.*;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sfn.SfnClient;
+import java.io.*;
+import java.util.*;
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class StepFunctionsTest {
+
+    private static  SfnClient sfnClient;
+    private static String stateMachineArn = ""; // gets dynamically set in a test
```

Comment:
// gets dynamically set in a test-> // Gets dynamically set in a test.

Reply:
fixed